diagnostic.md

```diff
@@ -6,70 +6,69 @@ Place your responses inside the fenced code-blocks where indicated by comments.
 What are two ways of telling rails where to route a GET request to '/books/4'?
 
 ```rb
-// your response here
+get '/books', to: 'books#index'
+resources :books, only: [:index]
 ```
 
 In what file does the above code go?
 
 ```md
-// your response here
+routes.rb
 ```
 
 In your own words, what does the command `bundle exec rake db:migrate` do?
 
 ```md
-// your response here
+It activates the migration files you have. It is going to change the schema on of project.
 ```
 
 What is the difference between `rails g scaffold things` and
 `rails g controller things`?
 
 ```md
-// your response here
+Controller is part of what is created with scaffold. Scaffold is going to generate everything you need on the MVC structure. (Model, View and Controller).
 ```
 
 What is the job of the `serializer`?
 
 ```md
-// your response here
+To make tables communicate with themselves. The serializer provides the id necessary to find the relationship (one-to-one, one-to-many) between information in 2 different tables.
 ```
 
 What do we expect the command `Patient.all` to return?
 
 ```md
-// your response here
+A list of instance objects of Patient class.
 ```
 
 We use the constant `Patient` in the `PatientsController`, but where is it
 actually defined?
 
-```md
-// your response here
+In the Model.
 ```
 
 Why do our `model` files appear to be lacking so much code, when they're doing
 so much for us?
 
 ```md
-// your response here
+As of now there is no block of code being executed on the Model. The Model is used to execute the code, before sending data back to the controller. If there is no code, it is going to return all the information in the table.
 ```
 
 What does typing `bin/rails dbconsole` in a directory that houses a rails app do for
 us?
 
 ```md
-// your response here
+It opens a psql environment where the tables of that app are stored.
```

Comment:
I believe that only the second command will actually open the `psql` environment